static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -284,7 +284,7 @@ CStudioAuthoring.Module.requireModule(
           templates = rteConfig.templates && rteConfig.templates.template ? rteConfig.templates.template : null;
 
           // https://www.tiny.cloud/docs/plugins/
-          pluginList = rteConfig.plugins;
+          pluginList = rteConfig.plugins + ' paste';
```

Comment:
What if it's already there?
Also, add comment stating why we are hard coding paste plugin.

Reply:
It doesn't affect if plugin also comes from configuration file.
Comment added